website/docs/r/node.html.markdown

```diff
@@ -30,9 +30,12 @@ The following arguments are supported:
 * `name` - (Required) The name of the node being added to, or
   referenced in the catalog.
 
+* `node_meta` - (Optional) KV metadata pairs
+
 ## Attributes Reference
 
 The following attributes are exported:
 
 * `address` - The address of the service.
 * `name` - The name of the service.
+* `node_meta` - (https://www.consul.io/docs/agent/http/catalog.html#Meta) KV metadata pairs
```

Comment:
I don't think this link works. I would just word this the same as the above:

> `node_meta` – Key/value pairs that are associated with the node.

Reply:
Fixed